app/src/main/java/com/example/ithaca_transit_android_v2/Models/Trip.kt

```diff
@@ -1,6 +1,7 @@
 package com.example.ithaca_transit_android_v2.Models
 
 import com.example.ithaca_transit_android_v2.models.Location
+import java.util.Date
```

Comment:
is this import necessary?

Reply:
Yes, had to import it because one of the parameters of [trip] is a date object